app/src/main/java/com/example/fitappa/View/MainActivity.java

```diff
@@ -10,16 +10,20 @@
 import com.google.firebase.auth.FirebaseUser;
 import com.google.firebase.firestore.FirebaseFirestore;
 
-import java.util.Optional;
-
 public class MainActivity extends AppCompatActivity {
 
+    /**
+     * This method is called when the activity starts.
+     * @param savedInstanceState contains the data it was most recently supplied with by onSaveInstanceState
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
 
+
         // TODO: refactor and make this if statement be a method call in a gateway
+        // TODO: Turn this into a private method - Abdullah
```

Comment:
Refactor into private method and remove return

Reply:
Refactored, please review again